.changeset/wet-dots-pretend.md

```diff
@@ -0,0 +1,5 @@
+---
+'faustwp': patch
+---
+
+Normalize sitemap URLs in order to support 'partially headless' sites.
```

Comment:
Without any context I might wonder what exactly "normalize" means. Could we expand on that a little so it's easier for a user to asses the potential impact of this change? Maybe something like:

> Ensure sitemap URLs use the WordPress domain and not the headless frontend domain. Fixes a conflict with Yoast SEO that prevented post links from being added to the posts sitemap.

Reply:
Wrote a commit message instead of changelog entry 😄